api/lambdas/sparql.test.ts

```diff
@@ -0,0 +1,148 @@
+import { APIGatewayProxyResult } from 'aws-lambda';
+import * as nock from 'nock';
+import { handler } from './sparql';
+
+describe('post-sparql', () => {
+  beforeAll(() => {
+    nock.disableNetConnect();
+
+    process.env['SPARQL_URL'] = 'http://sparql.local/sparql';
```

Comment:
This is an interesting way to set env vars. If you look at `develop` you'll see a `.jest/setEnvVars.js` file which is how I learned to do this (and the first answer if you Google this). Do you have an opinion on one vs the other?

Reply:
I see that you modify this env var - so maybe the answer is "both are acceptable depending on what you need to do with that variable".

Reply:
My preference is to set them here in the test for a couple of reasons. For one thing, it makes it explicitly clear what environment variables we're testing. The `.jest/setEnvVars.js` file is a little more magical in the sense that it's happening behind the scenes. For what it's worth, I don't actually see that file on the `develop` branch. I also like that this is _only_ setting the env var that I care about. It looks like the `.jest/...` approach would be setting the same env vars for all of the test files, whether they cared about those values or not.